pkg/gatherers/conditional/gather_alertmanager_logs.go

```diff
@@ -0,0 +1,127 @@
+package conditional
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/klog/v2"
+
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift/insights-operator/pkg/gatherers/common"
+	"github.com/openshift/insights-operator/pkg/record"
+
+	"github.com/openshift/insights-operator/pkg/gatherers"
+)
+
+// BuildGatherAlertmanagerLogs collects alertmanager logs for pods firing one the configured alerts.
+//
+// * Location in archive: conditional/namespaces/<namespace>/pods/<pod>/containers/<container>/logs/alertmanager-<event-name>.log
+// * Since versions:
+//   * 4.10+
```

Comment:
Most enhancements/gatherers also include the "Id in config" (internal code-name of the gatherer) and occassionally the path of a sample file in the sample archive. Just something to consider adding.